src/components/events/commonEventFields/additionalDetails.tsx

```diff
@@ -40,7 +40,7 @@ const AdditionalDetails = ({
           <TextField
```

Comment:
`    {details.map((attr :keyof IEvent) => (`

Reply:
Done in another way